profiles/base.yaml

```diff
@@ -1800,32 +1800,14 @@ spell_cycle:
 
 restock_shop:
 
-#custom almanac noun
+# The noun of your almanac (e.g. almanac, tome, treatise, monograph, etc)
 almanac_noun: almanac
-# For almanacs that let you pick which skill to use
-almanac_skills:
 
-sanowret_adjective: sanowret
-sanowret_no_use_scripts: # Don't try to use crystal while these scripts are active.
```

Comment:
Despite what GitHub says, these lines were not "deleted", they simply were moved below the new almanac setting